src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.html

```diff
@@ -1,164 +1,240 @@
-          </td>
-          <td>
-            <div class="center-block pie">
-              <cd-health-pie [data]="contentData"
-                             (prepareFn)="prepareRawUsage($event[0], $event[1])"></cd-health-pie>
-            </div>
-          </td>
-          <td>
-            <div class="center-block pie">
-              <cd-health-pie [data]="contentData"
-                             (prepareFn)="preparePoolUsage($event[0], $event[1])"></cd-health-pie>
-            </div>
-          </td>
-        </tr>
-        <tr>
-          <td i18n>Objects</td>
-          <td>
-            <ng-container i18n>Raw capacity</ng-container>
-            <br>
-            <ng-container i18n="disk used">({{ contentData.df.stats.total_used_bytes | dimlessBinary }} used)</ng-container>
-          </td>
-          <td i18n>Usage by pool</td>
-        </tr>
-      </table>
+    <cd-info-card cardTitle="Monitors"
+                  link="/monitor"
+                  class="col-md-3"
+                  contentClass="content-highlight"
+                  *ngIf="contentData.mon_status">
+      {{ contentData.mon_status | monSummary }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="OSDs"
+                  link="/osd"
+                  class="col-md-3"
+                  contentClass="content-highlight"
+                  *ngIf="contentData.osd_map">
+      <span *ngFor="let result of (contentData.osd_map | osdSummary)" [ngStyle]="result.style">
+        {{ result.content }}
+      </span>
+    </cd-info-card>
+
+    <cd-info-card cardTitle="Manager Daemons"
+                  class="col-md-3"
+                  contentClass="content-highlight"
+                  *ngIf="contentData.mgr_map">
+      {{ contentData.mgr_map | mgrSummary }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="Hosts"
+                  link="/hosts"
+                  class="col-md-3"
+                  contentClass="content-medium content-highlight">
+      {{ contentData.hosts }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="Object Gateways"
+                  link="/rgw/daemon"
+                  class="col-md-3"
+                  contentClass="content-medium content-highlight">
+      {{ contentData.rgw }}
+    </cd-info-card>
+    <cd-info-card cardTitle="Metadata Servers"
+                  class="col-md-3"
+                  contentClass="content-highlight"
+                  *ngIf="contentData.fs_map">
+      {{ contentData.fs_map | mdsSummary }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="iSCSI Gateways"
+                  link="/block/iscsi"
+                  class="col-md-3"
+                  contentClass="content-medium content-highlight">
+      {{ contentData.iscsi_daemons }}
+    </cd-info-card>
+  </cd-info-group>
+
+  <cd-info-group groupTitle="Performance"
+                 class="row info-group">
+
+    <cd-info-card cardTitle="Client IOPS"
+                  class="col-5"
```

Comment:
I would use this class only for big screens.
If you have a md screen or even a lg screen, the charts are too small.

My suggestion would be:
normal: "col-xs-12 col-sm-6 col-md-4 cd-col-5"
charts:   "col-xs-12 col-sm-6 col-md-6 cd-col-5"
logs: "col-md-12 col-lg-6"

col-5 would have a media query with min-width 1200px.

Reply:
Logs: OK.
Medium cards (including charts): bootstrap classes do not play well with 5 cols (awkward look) so I adapted cd-col-5 with media queries to show  bigger charts
for md screen and lg screen.